package.json

```diff
@@ -2,19 +2,34 @@
   "name": "obj-multiplex",
   "version": "1.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "node test"
+    "build": "tsc --project .",
+    "lint": "eslint . --ext ts,js,json",
+    "lint:fix": "yarn lint --fix",
+    "test": "yarn build && node test"
```

Comment:
Optional: I prefer to have the `test` _not_ include the `build` step as well, because it's annoying to have to wait for a rebuild when you just want the tests to run (e.g. when you're rewriting the tests).

That said, it's easy to forget to run the `build` script when you've changed the code. I leave it up to you to decide!

Reply:
As you mentioned it is easy to miss running the `build` script after a code change and that unintentional slip up can waste a lot of time if unnoticed, and since the update frequency of tests is less than the code changes for the functionality or bug fixes.. I think keeping the `build` script with the `test` will be a good idea.